docs/podman-derivative-api.md

```diff
@@ -0,0 +1,44 @@
+# How to use libpod for custom/derivative projects
+
+libpod today is a Golang library and a CLI.  The choice of interface you make has advantages and disadvantages.
+
+Running as a subprocess
+---
+
+Advantages:
+
+ - Many commands output JSON
+ - Works with languages other than Golang
+ - Easy to get started
+
+Disadvantages:
+
+ - Error handling is harder
+ - May be slower
+ - Can't hook into or control low-level things like how images are pulled
+
+Vendoring into a Go project
+---
+
+Advantages:
+
+ - Significant power and control
+ 
+Disadvantages:
+
+ - You are now on the hook for container runtime security updates (partially, `runc` is separate)
```

Comment:
perhaps s/for instance `runc` and `crun` are separate/`runc` is separate/

Reply:
Fixed, thanks!